Sources/SwiftBuildSupport/SwiftBuildSystem.swift

```diff
@@ -700,6 +707,16 @@ public final class SwiftBuildSystem: SPMBuildCore.BuildSystem {
         }
     }
 
+    public func writePIF(buildParameters: BuildParameters) async throws {
```

Comment:
suggestion (blocking): looking at the API in isolation without looking at the implementation, there is nothing that provide where the PIF is being written to.  As consumer of the API, how can we get the location of where the PIF was written?

The API should be update to either accept a path where to write the PIF or maybe a return either (1) the path where the PIF, or (2) the pif contents.